include/seastar/http/client.hh

```diff
@@ -76,6 +76,10 @@ class connection : public enable_shared_from_this<connection> {
     hook_t _hook;
     future<> _closed;
     internal::client_ref _ref;
+    // Connections are persistent by default in HTTP-1.1 and above, so it's
+    // not 100% compatible with older versions. Since such servers are very
+    // likely not in use nowadays, keep it like this for simplicity
```

Comment:
Since Seastar is a library, personally I would have liked to see 100%-accurate implementations for all cases (even obscure ones) rather than comments about not being 100%, especially since we're likely to ever review this area of the code again in the future, but I don't insist - I agree that the chance of finding an HTTP-1.0-only server in 2023 is 0%, and probably was 0% already a decade ago.

Reply:
OK, updated it like that